lib/src/screens/dashboard/widgets/address_page.dart

```diff
@@ -60,16 +60,19 @@ class AddressPage extends BasePage {
   bool effectsInstalled = false;
 
   @override
-  Color get titleColor => Colors.white;
+  Color? get titleColor =>
+      currentTheme.type == ThemeType.bright ? Colors.white : null;
```

Comment:
can be removed since we are overriding the leading widget entirely and `titleColor` is only used there